Document/0x08a-Testing-Tools.md

```diff
@@ -12,8 +12,10 @@ title: Testing Tools
     - Open-source
     - Free to use
     - Capable of analyzing recent Android/iOS applications
-    - Regularly updated
+    - Regularly updated or stable by nature
```

Comment:
To avoid maintaining this text in 2 places, maybe you can extract it from here and leave a link to "Can my tool be added" in the contributing file.